lib/adapters/apply-edit-adapter.ts

```diff
@@ -1,7 +1,19 @@
 import type * as atomIde from "atom-ide-base"
 import Convert from "../convert"
-import { LanguageClientConnection, ApplyWorkspaceEditParams, ApplyWorkspaceEditResponse } from "../languageclient"
+import {
+  LanguageClientConnection,
+  ApplyWorkspaceEditParams,
+  ApplyWorkspaceEditResponse,
+  WorkspaceEdit,
+  TextDocumentEdit,
+  CreateFile,
+  RenameFile,
+  DeleteFile,
+  DocumentUri,
+} from "../languageclient"
 import { TextBuffer, TextEditor } from "atom"
+import * as fs from "fs"
```

Comment:
This cleans up deep calls to `fs.promises.*`
```suggestion
import { promises as fsp, Stats } from "fs"
```